client/nomad.go

```diff
@@ -27,6 +37,43 @@ func NewNomadAPI() *APIClient {
 	return NewAPIClient("nomad", addr, headers)
 }
 
+// NewNomadTLSConfig returns a *TLSConfig object, using
+// default environment variables to build up the object.
+func NewNomadTLSConfig() (*TLSConfig, error) {
+	tlsConfig := TLSConfig{}
+	if v := os.Getenv(EnvNomadCaCert); v != "" {
+		tlsConfig.CACert = v
+	}
```

Comment:
Thanks. I appreciate the gut reaction! I added a response to a previous comment about the map idea, so I'll skip it here. Hopefully the more concise TLSConfig struct building helps to alleviate *some* of the concerns that would otherwise be handled by the map.